README.md

```diff
@@ -131,3 +123,30 @@ To extract metadata from an affiliation string:
 	$ curl -X POST --data "affiliation=the text of the affiliation" \
 	  http://cermine.ceon.pl/parse.do
 
+
+(OPTIONAL) Build & Run from sources
+-------------
+
+If you would like to build an executable JAR yourself, clone the project and follow these steps.
+
+**Preconditions**
+
+- Ensure that Apache Maven is installed (`sudo apt install maven`)
+
+**Build **
```

Comment:
Please remove this additional space, it seems to break the layout: https://github.com/philonor/CERMINE/blob/5c2790f9fd1dc3a087fef698c97bb2663a9a0974/README.md